dataland-backend/src/main/kotlin/org/dataland/datalandbackend/interfaces/DataStoreInterface.kt

```diff
@@ -23,9 +26,37 @@ interface DataStoreInterface {
     fun listDataSets(): List<DataSetMetaInformation>
 
     /**
-     * Method to get the values of a single data set
-     * @param id identifier of the stored data set
+     * Method to get the data of a single entry in the data store
+     * @param dataIdentifier identifier of the stored data (consists of data id and data type)
      * @return all data associated to the identifier provided
      */
-    fun getDataSet(id: String): DataSet
+    fun getCompanyAssociatedDataSet(dataIdentifier: DataIdentifier): List<String>
```

Comment:
Hier eine List<String> zurückzugeben, wo das 0-te Element ein json-String ist, und das 1-te Elemente eine company-id halte ich für nicht gut in einer Typisierte Sprache. Das ist dann so magic ("man muss in dem Return nur an stelle 93 nachsehen, da steht dann schon drin was man will"). 

Warum returnen wir nicht einfach storableDataSet?

Reply:
ok